### DISCUSSION TITLE
Will we hava a tatol independent desktop app?

### DISCUSSION THREAD

**Discussion by Mojicode** [toeverything/AFFiNE#542] [General]
I don't want use docker with browser，i  don't hava a server machine. Do we hava a plan to make a independent app like other desktop app?

**Comment by ShortCipher5** [comment]
No fixed timeline, but it's definitely something we'll be considering - maybe early 2023 for some official native clients. There are currently some unofficial community contributed clients you may want to check out and play with in the mean time.

**Comment by tzhangchi** [comment]
We are building AFFiNE Client App , you can watch the branch https://github.com/toeverything/AFFiNE/tree/feat/client-app  @Mojicode

**Reply by Mojicode** [reply]
> https://github.com/toeverything/AFFiNE/tree/feat/client-app

The branch has been gone

**Reply by doodlewind** [reply]
The client code is merged into https://github.com/toeverything/AFFiNE/tree/master/apps/desktop now. Need some time to fine-tuning it, though.

**Comment by HeJiachen-PM** [comment]
Archieved
